core/core_bind.cpp

```diff
@@ -373,6 +373,10 @@ Dictionary _OS::get_time(bool utc) const {
  * @return epoch calculated
  */
 int64_t _OS::get_unix_time_from_datetime(Dictionary datetime) const {
+
+	// if datetime is an empty Dictionary throws an error
```

Comment:
The first empty line should be removed to match the style guide (for `master` - in `3.2` we did use to have such empty lines at the start of functions but we changed that style).

Reply:
Please do the changes with `git commit --amend` to avoid creating a new commit for this style fix. (And then you can force push to update your branch and this PR.)

Reply:
sure!

Reply:
done

Reply:
note that this cannot be cherry-picked to current 3.2.4 because `dictionary.is_empty()` in `master` is `dictionary.empty()` on that release.

Reply:
That's fine, I can fix that when cherry-picking.